docs/en/basic/Command-Line-Interface.md

```diff
@@ -23,94 +23,93 @@ Usage: alluxio [COMMAND]
 
 ## General operations
 
-In this section, we list the usages and examples of general Alluxio operations except file system 
-and file system admin operations. File system admin operations are listed in the [Admin CLI
-doc]({{site.baseurl}}{% link en/operation/Admin-CLI.md %}).
+This section lists usages and examples of general Alluxio operations with the exception of file
+system commands which are covered in the [Admin CLI doc]({{site.baseurl}}{% link en/operation/Admin-CLI.md %}).
 
 ### format
 
-The `format` command will format the Alluxio master and all workers. 
+The `format` command formats the Alluxio master and all its workers. 
 
-If you run this command for an existing Alluxio cluster, all data and metadata stored in Alluxio will be deleted. 
-However, data in under storage will not be changed.
+TODO: this doesn't make sense; adding -s will only run format if the UFS is local? or does -s also format the UFS assuming it is local and not exists? 
```

Comment:
real solution is to get rid of the `-s` option because it makes no sense to begin with. What makes no sense cannot be sensibly explained. Skipping this todo for now